### ISSUE TITLE
FC2 Feedback:  if you could zebra-stripe every other results row's background color (alternating between white and very light grey, for example) that would make it easier to read

### ISSUE THREAD

**Issue by DeniseColbert** [FreeUKGen/FreeCENMigration#325]
Right now the rows do light up with a different background color if you mouseover them, but I think static striping would be better.

**Comment by AlOneill** [comment]
We already have striped table style available. The colour would probably need overriding — currently pale yellow — but that could create another issue ...

The striped style does not remove the row highlighting: this style would have to be written out if we stripe with light grey (and white).

**Comment by PatReynolds** [comment]
I don't see stripes of any colour (chrome or comodo). do we need testing on other browsers?

**Comment by AlOneill** [comment]
@PatReynolds Yes, the striping is available, but we don't use it currently. The colour is currently a pale yellow: this is what it **would** look like, here on a Search Results table (using the new colour scheme):

![style_gallery___freereg](https://user-images.githubusercontent.com/14852554/34457579-f5fcb736-edac-11e7-936c-b603bb71ab0c.png)

We can alter the colour to suit, naturally. To be unambiguous on all the sites it would be best to avoid all of our project colours (teal, magenta, lavender, turquoise), which leaves the pale yellow/orange part of the spectrum to choose from, plus pale grey.

The row highlighting will continue to be effective, unless we choose a grey to stripe with: or we change the highlighting colour. The row-highlighting styling is pure inuit: it cannot be eradicated, but we could effectively disable it by setting the highlight colour to transparent/white.

**Comment by PatReynolds** [comment]
I'm happy with primrose (the colour you have here) for the stripes.  Probably need to test mock ups of this vs no stripes with users.

**Comment by AlOneill** [comment]
Try looking at those yellow stripes for any length of time — it did my nut in!!

Having spent some time experimenting with stripe colours, as opposed to theorising(!), I concluded that those nearer to grey (i.e. more neutral) than the yellow were more comfortable. For example:

![style_gallery___freereg](https://user-images.githubusercontent.com/14852554/34463313-cbc646f0-ee4f-11e7-81c1-cd7dc1d3d6ce.png)

However, light grey is the most comfortable of them all, but then ...

![style_gallery___freereg](https://user-images.githubusercontent.com/14852554/34463398-8f55eaf6-ee52-11e7-9cfe-b6bd7569c122.png)

If we are inclined to use static striping for some Researcher tables, then I suggest that striping should be part of a comprehensive review of table styling, both for Researchers and Members. In other words, take this user's expression of dissatisfaction seriously, but not necessarily their 'solution' — they may know more than us about UI design, but my guess is not. For example, I bet that centred content in table cells causes more problems with scanning entries than does a lack of striping — after all, we do use cell borders. (I am aware that this para is not worded very well, but I hope you get the gist — it would be unwise to consider striping in isolation.)

**Comment by AlOneill** [comment]
Apologies for using FreeREG example for FreeCEN issue. But ...

**Comment by PatReynolds** [comment]
I completely agree!

**Comment by richpomfret** [comment]
To continue reviewing - perhaps to roll out as a tet with the zebra suggestion, or finding an alternative solution based on Alison's comments/thoughts.

**Comment by PatReynolds** [comment]
Something similar is being implemented on FreeBMD2, and could be used on other projects.
